justfile

```diff
@@ -210,4 +211,12 @@ wait-for-electrs-to-be-ready:
         fi
     done
 
+build-ipa:
+    cd mobile && flutter build ipa --dart-define="ELECTRS_ENDPOINT=${ELECTRS_ENDPOINT}" --dart-define="COORDINATOR_P2P_ENDPOINT=${COORDINATOR_P2P_ENDPOINT}"
+
+publish-testflight:
+    cd mobile && xcrun altool --upload-app --type ios --file ./build/ios/ipa/10101.ipa --apiKey ${ALTOOL_API_KEY} --apiIssuer ${ALTOOL_API_ISSUER}
+
+release-testfligth: gen ios build-ipa publish-testflight
```

Comment:
nit: typo `flight` instead of `fligth` (here and in readme).

you can probably shorten this to `TestFlight` and add a comment in the line above `# build an iOS binary and release to TestFlight` (when you query `just --list`, then the comments appear too!)